.gitignore

```diff
@@ -555,3 +555,6 @@ MigrationBackup/
 
 # End of https://www.gitignore.io/api/node,komodoedit,webstorm+all,visualstudio,visualstudiocode
 push-docker.ps1
+
+# asdf
```

Comment:
?

Reply:
i use [asdf](https://asdf-vm.com/) to manage my node versions during local dev, so i ignored the config file it generates per repo. I can add a link to this comment to make it more clear